Gems/Atom/RPI/Assets/Shaders/ImagePreview.azsl

```diff
@@ -89,5 +91,8 @@ PSOutput MainPS(VSOutput psInput)
             break;
         }
     }  
+
+    OUT.m_color.xyz = (OUT.m_color.xyz - float3(PassSrg::m_colorRangeMinMax[0],PassSrg::m_colorRangeMinMax[0],PassSrg::m_colorRangeMinMax[0])) / (PassSrg::m_colorRangeMinMax[1] - PassSrg::m_colorRangeMinMax[0]);
```

Comment:
Spaces after the commas and a comment would make this a lot more readable 🙂

Reply:
btw, you can also do
PassSrg::m_colorRangeMinMax[0].xxx
If you want to create a float3 out of a single float:
`OUT.m_color.xyz = (OUT.m_color.xyz - PassSrg::m_colorRangeMinMax[0].xxx) / (PassSrg::m_colorRangeMinMax[1] - PassSrg::m_colorRangeMinMax[0]);`

Reply:
Right, thanks. I was moving fast and forgot to clean this up. The xxx thing escaped me at the  moment too, thanks for the reminder.

Reply:
You can subtract scalars from vectors, shouldn't need the `.xxx` at all I believe